deck-o-cards.js

```diff
@@ -9,41 +9,52 @@ function deck_o_cards() {
 
   // Make 52 card objects and store them in the "cards" array
   // Hint: use 2 for loops
+  for (var i = 0; i < values.length; i++) {
+    for (var k = 0; k < suits.length; k++) {
+      var singleCards = { Suit: suits[k], Value: values[i] };
```

Comment:
Name this something singular since it's only one object (i.e. singleCard). The array can be named something plural since its a list.